akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -57,7 +54,7 @@ Scala
 Java
 :  @@snip [MyActor.java]($code$/java/jdocs/actor/MyActor.java) { #imports #my-actor }
 
-Please note that the Akka Actor `receive` message loop is exhaustive, which
+Please note that the Akka Actor `AbstractActor.Receive` message loop is exhaustive, which
```

Comment:
This section is actually not true anymore with the receive builder, it builds an actual PartialFunction and works just like Scala.

Reply:
I see, will remove `receive` reference than as there is no such in Java